docs/source/models/cause.rst

```diff
@@ -337,14 +336,38 @@ Mortality Impacts
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The cause-specific mortality rate is the mortality rate from a specified cause for a population. 
+The numerator is the number of deaths attributed to a specific cause. The denominator remains the 
+size of the population. In GBD, cause-specific estimates by country, year, age, sex.
+
+  For example, 112 measles deaths in country A during calendar year 2016. 1010,623 estimated 2016 mid-year 
```

Comment:
@alibow make sure the formatting is consistent here with what you've been doing.

Reply:
This part looks good, although I've also been adding a second paragraph to the example portion that explains how this rate is used in the model. (See @collijk's suggestions below)